knowledge graph/kg_agent.py

```
@@ -0,0 +1,351 @@
+            
+            # create the entity
+            result = await session.run(
+                f"CREATE (e:{entity_type} {{{props_string}}}) RETURN e",
+                **properties
+            )
+            record = await result.single()
+            return f"Created {entity_type} entity with ID: {record['e'].id}"
+    
+    async def get_entity(self, entity_type: str, properties: Dict[str, Any]):
+        """Get an entity from the knowledge graph"""
+        _validate_label(entity_type)
```

Ruff (0.12.2)

Undefined name `_validate_label`

(F821)